packages/app/src/app/components/EditableTags/index.js

```diff
@@ -58,10 +67,9 @@ export default function(color) {
   }
 `;
 
-  return props => (
-    <Fragment>
-      <GlobalStyle />
-      <TagsInput {...props} />
-    </Fragment>
-  );
-}
+export default props => (
+  <Fragment>
+    <GlobalStyle color={props.template.color()} />
```

Comment:
I think you can put this in, with this you don't have to do `hexToRgb` of `color` in the tag.

```suggestion
    <GlobalStyle color={props.template.color} />
```